Sources/Matrix3x3+nosimd.swift

```diff
@@ -148,12 +148,14 @@ public struct Matrix3x3f {
             - (m13 * m22 * m31 + m11 * m23 * m32 + m12 * m21 * m33)
     }
     
-    public var transpose: Matrix3x3f {
-        return Matrix3x3f(m11, m21, m31, m12, m22, m32, m13, m23, m33)
+    public var transposed: Matrix3x3f {
+        return Matrix3x3f(m11, m12, m13, 
```

Comment:
it is really uncomfortable for me every time to remember if we do column or row-major haha

Reply:
okay it seems correct

Reply:
yea, it is confusing here. I suggest to rewrite matrix3x3 and matrix4x4